examples/04_colors_and_texture.py

```diff
@@ -6,7 +6,7 @@
 
 import moderngl
 import numpy as np
-from objloader import Obj
+from moderngl_obj import Obj
```

Comment:
`moderngl_obj` is deprecated and `objloader` should be used instead.

Reply:
I reverted that further down when I checked the age of the packages on pipy